policies/recipes/steps.go

```diff
@@ -144,26 +143,41 @@ func StepScriptRun(step *osconfigpb.SoftwareRecipe_Step_ScriptRun, artifacts map
 		return err
 	}
 
-	var qargs []string
-	//if step.ScriptRun.Interpreter == osconfigpb.SoftwareRecipe_Step_RunScript_POWERSHELL {
-	for _, arg := range step.ScriptRun.Args {
-		qargs = append(qargs, fmt.Sprintf("%q", arg))
+	var command string
+	var args []string
+	switch step.ScriptRun.Interpreter {
+	case osconfigpb.SoftwareRecipe_Step_RunScript_POWERSHELL:
+		command = "PowerShell.exe"
```

Comment:
Use the full path here (C:\Windows\System32\WindowsPowerShell\v1.0\powershell.exe), you might even want to use the SystemRoot environment variable to grab the correct Windows directory but that may be overkill

Also do we want to have a check for Windows or note here? Otherwise the error would be odd.

Reply:
the error will be that powershell.exe isn't found, right?

Reply:
That's an obtuse error for the user though, a simple check would allow us to produce a much clearer error "PowerShell interpreter not supported on non Windows systems" or something like that.